Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveM.cpp

```diff
@@ -119,8 +120,16 @@ void FiniteDifferenceSolver::MacroscopicEvolveM (
               // keep the interpolation. The IntVect is (1,0,0) to interpolate values to the x-face.
               Real mag_gamma_interp = MacroscopicProperties::macro_avg_to_face(i,j,k,amrex::IntVect(1,0,0),mag_gamma_arr);
               Real Gil_damp = PhysConst::mu0 * mag_gamma_interp
```

Comment:
Maybe change the logic to look like:

`Real Ms = (M_normalization == 0) ?
std::sqrt( std::pow(M_xface(i, j, k, 0),2.0) + std::pow(M_xface(i, j, k, 1),2.0) + std::pow(M_xface(i, j, k, 2),2.0) ) :
MacroscopicProperties::macro_avg_to_face(i,j,k,amrex::IntVect(1,0,0),mag_Ms_arr);

Real Gil_damp = PhysConst::mu0 * mag_gamma_interp * MacroscopicProperties::macro_avg_to_face(i,j,k,amrex::IntVect(1,0,0),mag_alpha_arr) / Ms
`